DuckDuckGo/SettingsSubscriptionView.swift

```diff
@@ -93,7 +95,9 @@ struct SettingsSubscriptionView: View {
                                  subtitle: UserText.settingsPProDBPSubTitle,
                                  action: { isShowingDBP.toggle() }, isButton: true)
                 .sheet(isPresented: $isShowingDBP) {
-                    SubscriptionPIRView()
+                    SubscriptionPIRView().onAppear(perform: {
+                        Pixel.fire(pixel: .privacyProPersonalInformationRemovalSettings)
```

Comment:
I'm not sure about adding non-view related event here.  We should probably fire the pixel from the ViewModel.   Also all pixel-related events that this view needs to fire could be grouped into a single method there.

Reply:
Same for this view.  It's a single point of entry.

Reply:
Done